docs/source/release_notes.rst

```diff
@@ -3,15 +3,23 @@
 Release Notes
 -------------
 
+Breaking Changes
```

Comment:
the Breaking Changes section should go between the Future Release section and the most recent release